tests/foreman/cli/test_ansible.py

```diff
@@ -48,11 +49,17 @@ def test_positive_ansible_e2e(target_sat, module_org, rhel_contenthost):
 
     :CaseAutomation: Automated
 
+    :BZ: 2154184
+
+    :customerscenario: true
+
     :CaseImportance: Critical
     """
     SELECTED_ROLE = 'RedHatInsights.insights-client'
     SELECTED_ROLE_1 = 'theforeman.foreman_scap_client'
     SELECTED_VAR = gen_string('alpha')
+    # disable batch tasks to test BZ#2154184
+    Settings.set({'name': "foreman_tasks_proxy_batch_trigger", 'value': "false"})
```

Comment:
Should this perhaps be parametrized?

Reply:
I guess that @lhellebr's intent was to put it into an `is_open` logical block to ensure that we do not apply the workaround after the BZ is fixed.

Reply:
the bug is fixed, this is not a workaround but a way to cover for that specific scenario

Reply:
@pondrejk I think by parametrize @lhellebr meant, shouldn't we reset this setting back to default in teardown?